src/tests/unit/services/UserInfoRequestProcessor.test.ts

```diff
@@ -334,15 +333,15 @@ describe("UserInfoRequestProcessor", () => {
 		const out: Response = await userInforequestProcessorTest.processRequest(VALID_USERINFO);
 		expect(mockCicService.getSessionById).toHaveBeenCalledTimes(1);
 		expect(mockCicService.getPersonIdentityBySessionId).toHaveBeenCalledTimes(1);
-		expect(mockCicService.sendToTXMA).toHaveBeenCalledTimes(1);
+		expect(mockCicService.sendToTXMA).toHaveBeenCalledTimes(2);
```

Comment:
Could we please add a test that checks `expect(mockCicService.sendToTXMA).toHaveBeenCalledWith()` to make sure that the calls have the right details included?

Reply:
Done above, didn't both for the failure scenarios